packages/neos-ui-redux-store/src/CR/Workspaces/selectors.js

```diff
@@ -14,3 +14,21 @@ export const publishableNodesInDocumentSelector = createSelector(
     ],
     (publishableNodes, activeDocumentContextPath) => publishableNodes.filter(i => $get('documentContextPath', i) === activeDocumentContextPath)
 );
+
+export const makeIsDocumentNodeDirtySelector = documentContextPath => createSelector(
```

Comment:
this selector still needs to be rewritten (to same way as the selector below  regarding arguments :) )